noaa/README.txt

```diff
@@ -0,0 +1,5 @@
+Dataset containing daily weather measurement from NOAA:
+ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/by_year/
+
+The dataset has been processed by: https://gist.github.com/colings86/078e85a1131324471f4f10c73570d678
```

Comment:
Can you just compress the zip file from the gist and dump it here so it is self-contained? Also, the gist contains instructions, especially: 

>Sort files using something like sort --field-separator=',' --key=1,2 -o ~/Downloads/2017-sorted.csv ~/Downloads/2017.csv

And I think you should document how you sorted the files.